### ISSUE TITLE
Helm: spec.ports: Required value

### ISSUE THREAD

**Issue by zerok** [gomods/athens#1228]
When I try to install the gomods/athens-proxy helm chart in version 0.2.5 I get the following error message:

```
$ helm install --name athens --namespace athens --version 0.2.5  gomods/athens-proxy
Error: release athens failed: Service "athens-athens-proxy" is invalid: spec.ports: Required value
```

Looking at the service.yaml everything looks fine, though. Any idea what could cause that?

**Environment (please complete the following information):**

```
$ helm version
Client: &version.Version{SemVer:"v2.14.0", GitCommit:"05811b84a3f93603dd6c2fcfe57944dfa7ab7fd0", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.14.0", GitCommit:"05811b84a3f93603dd6c2fcfe57944dfa7ab7fd0", GitTreeState:"clean"}
```

```
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"13", GitVersion:"v1.13.3", GitCommit:"721bfa751924da8d1680787490c54b9179b1fed0", GitTreeState:"clean", BuildDate:"2019-02-04T04:48:03Z", GoVersion:"go1.11.5", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"12", GitVersion:"v1.12.4", GitCommit:"f49fa022dbe63faafd0da106ef7e05a29721d3f1", GitTreeState:"clean", BuildDate:"2018-12-14T06:59:37Z", GoVersion:"go1.10.4", Compiler:"gc", Platform:"linux/amd64"}
```

OS: macOS 10.14.x

**Comment by marwan-at-work** [comment]
cc: @arschles

**Comment by zerok** [comment]
I think I found the problem. I'm currently working on a PR for that.

**Comment by glb** [comment]
Fixed by #1249 .

**Comment by zerok** [comment]
@glb Yes :)
